src/pages/MeetingSetup/MeetingRecap.jsx

```diff
@@ -1,14 +1,29 @@
-import React from 'react';
-import { useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { useNavigate, useLocation } from 'react-router-dom';
-import { createMeeting } from '../../redux/slices';
```

Comment:
Page goes blank because of error if you type in the link directly, I get the error:

`Uncaught TypeError: Cannot destructure property 'buddy' of 'location.state' as it is null.`